draft-ietf-ppm-dap.md

```diff
@@ -1205,20 +1199,46 @@ The Leader and Helper initialization behavior is detailed below.
 #### Leader Initialization {#leader-init}
 
 The Leader begins the aggregate initialization phase with the set of candidate
-report shares as follows:
+reports as follows:
 
 1. Generate a fresh AggregationJobID. The ID value MUST be unique within the
    scope of the corresponding DAP task.
 1. Decrypt the input share for each report share as described in
    {{input-share-decryption}}.
 1. Check that the resulting input share is valid as described in
    {{input-share-validation}}.
-1. Initialize VDAF preparation as described in {{input-share-prep}}.
 
-If any step invalidates the report share, the Leader removes the report share
-from the set of candidate reports. Once the leader has initialized this state
-for all valid candidate report shares, it creates an `AggregationJobInitReq`
-message for each Helper to initialize the preparation of this candidate set. The
+If any step invalidates the report, the Leader removes the report from the set
+of candidate reports. Next, the Leader initializes VDAF preparation by running
+the following procedure for each report, with function definitions from
+{{!VDAF}}:
```

Comment:
Let's add a TODO here describing the current state of affairs: We'll need to cut a draft of VDAF with the new API and update the references here.